WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadUtils.java

```diff
@@ -511,9 +512,9 @@ public void onClick(View v) {
                         snackbarButtonRes = R.string.button_view;
 
                         if (post.isPage()) {
-                            snackbarMessageRes = R.string.page_published;
+                            snackbarMessageRes = isFirstTimePublish ? R.string.page_published : R.string.page_updated;
                         } else if (userCanPublish) {
-                            snackbarMessageRes = R.string.post_published;
+                            snackbarMessageRes = isFirstTimePublish ? R.string.post_published : R.string.post_updated;
```

Comment:
I wonder if it makes sense to introduce new strings here. :thinking: Perhaps, to distinguish between updating a draft post / page with a published one. Maybe something like `"Published (post|page) updated"`, or `"Updated published (post|page)"`. Wdyt?

Reply:
That is a good idea @mkevins 👍 
It makes perfect sense to inform the user on this. At this point we have the following messages per case:
1. New post/page published -> `post_published` / `page_published`
2. Published post/page edited -> `post_updated` / `page_updated`
3. Draft post/page published -> `post_published` / `page_published`
4. Draft post/page updated -> [`editor_draft_saved_online`](https://github.com/wordpress-mobile/WordPress-Android/blob/c8f3f5dc35e39e23bd58cac39b39fa29bb68b5b7/WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadUtils.java#L500) 

I agree that the `"Published (post|page) updated"`/`"Updated published (post|page)"` would be more clear for case 2 and is an easy change (if I'm not missing some other corner case). I suggest to leave it as is for now thought since the same string is used in the [equivalent notification](https://github.com/wordpress-mobile/WordPress-Android/blob/02d13098ed55216215da2265ef391f6fbb128a98/WordPress/src/main/java/org/wordpress/android/ui/uploads/PostUploadNotifier.java#L325) and the original reported issue (internal ref `pb3aDo-Eu#comment-1181`) suggests using the `"Page Updates"` string.

It would be also nice to be able to distinguish between cases 1 and 3 but since `onPostUploadedSnackbarHandler` method does not have all the information needed it would probably make the implementation even more complex and escapes the scope of the original [issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/3018).

Reply:
Thanks for listing the other cases / scenarios as well. I agree, cases 1 and 3 would be harder to tackle (I'm not sure it's worth the compexity). Also, you raise a good point on the consistency with the notifications for case 2.